src/AcmExpandableWrapper/AcmExpandableWrapper.tsx

```diff
@@ -33,7 +36,14 @@ export const AcmExpandableWrapper = (props: AcmExpandableWrapperProps) => {
 
     return (
         <div className={classes.root}>
-            {headerLabel && <Title headingLevel="h4">{headerLabel}</Title>}
+            {headerLabel && (
+                <Title headingLevel="h4">
+                    {headerLabel}
+                    {headerLabel === 'Saved Searches' && (
```

Comment:
Shouldn't hard code this text in a component like this

Reply:
replaced with component property for count